src/components/form-version/def-dropdown.vue

```diff
@@ -11,7 +11,7 @@ except according to the terms contained in the LICENSE file.
 -->
 <template>
   <div class="form-version-def-dropdown btn-group">
-    <button :id="toggleId" type="button" class="btn btn-primary dropdown-toggle"
+    <button :id="toggleId" type="button" class="btn btn-default dropdown-toggle"
```

Comment:
It looks like this fades into the background in the Your Current Draft section of the form overview:

<img width="331" alt="Screen Shot 2021-04-28 at 11 27 35 AM" src="https://user-images.githubusercontent.com/5970131/116431821-1fba9300-a816-11eb-973c-28e56618fb27.png">